subsys/bluetooth/mesh/brg_cfg_srv.c

```diff
@@ -5,13 +5,296 @@
  */
 
 #include <zephyr/bluetooth/mesh.h>
+#include <zephyr/bluetooth/mesh/brg_cfg.h>
 #include "brg_cfg.h"
+#include "foundation.h"
+#include "subnet.h"
 
 #define LOG_LEVEL CONFIG_BT_MESH_MODEL_LOG_LEVEL
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_mesh_brg_cfg_srv);
 
+static void bridge_status_send(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx)
+{
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_SUBNET_BRIDGE_STATUS, 1);
+
+	bt_mesh_model_msg_init(&msg, OP_SUBNET_BRIDGE_STATUS);
+	net_buf_simple_add_u8(&msg, (uint8_t) bt_mesh_brg_cfg_enable_get());
```

Comment:
![Screenshot from 2024-08-06 08-49-02](https://github.com/user-attachments/assets/6b7a1ae6-b545-47cf-acf5-662d76983943)
In most of C compilers and versions it will work. However, as you can see, C specification doesn't guarantee to 100% that `true` is always 1. There are compilers with obsolete behavior.
Code that you implemented depends on presence or absence of obsolete compiler behavior. The source code that I suggest is **really** cross-platform and compiler independent.

Reply:
Thanks a lot for this.